doc/connecting-lab.md

```diff
@@ -65,48 +93,25 @@ scheduler-lava:
       - 'lava-broonie'
 ```
 
-
-### yaml pull request
-
-The final step is to submit a pull request to the `kernelci-pipeline` repository
-to add the lab configuration to the yaml file.
-For example, see the following [pull request](https://github.com/kernelci/kernelci-pipeline/pull/426).
-
-In details the pull request should add a new entry to the `runtimes` section of the configuration file:
-
-```yaml
-
-  lava-broonie:
-    lab_type: lava
-    url: 'https://lava.sirena.org.uk/'
-    priority_min: 10
-    priority_max: 40
-    notify:
-      callback:
-        token: kernelci-new-api-callback
-        url: https://staging.kernelci.org:9100
-
-```
-Where `lava-broonie` is the name of the lab, `lava` is the type of the lab, `url` is the URL of the lab, `priority_min` and `priority_max` are the priority range allowed to jobs, assigned by lab owner, and `notify` is the notification configuration for the lab.  The `callback` section contains the token description (name) and the URL of the pipeline instance LAVA callback endpoint.
-More details how LAVA callback and token works can be found in the [LAVA documentation](https://docs.lavasoftware.org/lava/user-notifications.html#notification-callbacks).
-
 ### Jobs and devices specific to the lab
 
-For testing it is better to add separate job and platforms specific to the lab.  For example, the following yaml file adds a job and a device type for the `lava-broonie` lab:
```

Comment:
Not relevant to this thread but I might forget it later: should we fold lab-specific jobs into generic ones after some time or just leave them as they are for tracing purposes?

Reply:
Yes, we can define generic jobs. In fact, it would be a good practice to not use job configuration in the job name such as lab name and platform. It would also make our data submission to KCIDB easier as that's how different CI systems name their jobs as well.